File: PHP Word Examples/BackgroundApiDemo/setBackgroundImage.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\BackgroundApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new BackgroundApi($configuration);

$name = "setBackgroundImage.docx";
$image_path = "input/Background.png";
$password = null;
$folder = "input";
$storage = null;
$dest_file_path = "output/setBackgroundImage.docx";
$result=$apiInstance->setBackgroundImage($name, $image_path,$dest_file_path, $folder, $storage, $password);

File: Python Word Examples/BackgroundApiDemo/setBackgroundImage.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.background_api.BackgroundApi(configuration)

filename = 'SetBackgroundImage.docx'
folder = 'input'
imgePath = "input/Logo.png"
destFilePath = "output/SetBackgroundImage.docx"

result = api.set_background_image(name=filename,folder=folder, imge_path=imgePath, dest_file_path=destFilePath)